README.md

```diff
@@ -2,92 +2,133 @@
+### History Network: Headers, Blocks, and Receipts
+
+The History Network facilitates on-demand retrieval of the history of the Ethereum chain.  This includes:
+
+- Headers
+- Block bodies
+- Receipts
 
-Normal clients will download every block header to construct the canonical chain. This is unreasonable for a "stateless" client. The "[double-batched merkle log accumulator](https://ethresear.ch/t/double-batched-merkle-log-accumulator/571)" is the mechanism that enables portal clients to achieve this goal, without requiring them to download every canonical header.
+The responsibility for storing this data should be evenly distributed across the nodes in the network.  Nodes must be able to choose how much history data they want to store.  The data is distributed in a manner that allows nodes to determine the appropriate nodes to query for any individual piece of history data.  
 
-When a portal client requests a certain header, the response includes two accumulator proofs. As long as the client maintains an up-to-date view of the chain tip (via the gossip network) it can use the proofs to validate a headers inclusion in the canonical chain. Then, the client can use the header to validate other data retrieved from the portal network.
+Participants to this network are assumed to have access to the canonical header chain.
 
-The portal network is designed to emulate the following [ETH protocol](https://github.com/ethereum/devp2p/blob/d3f6d6724ea587fc28d3c22911d8ac0490627c8c/caps/eth.md#protocol-messages) messages, by supporting the following data to be retrieved via `block_hash`.
+All data retrieved from the history network is addressed by block hash.  Headers retrieved from this network can be validated to match the requested block hash.  Block Bodies and Receipts retrieved from this network can be validated against the corresponding header fields.
 
-| ETH Protocol | Portal Network |
-| - | - |
-| `GetBlockHeaders` -> `BlockHeaders` | `block_hash` -> header & inclusion proof |
-| `GetBlockBodies` -> `BlockBodies` | `block_hash` -> block body |
-| `GetReceipts` -> `Receipts` | `block_hash` -> block receipts |
 
-### Canonical Indices: Transactions by Hash and Blocks by Number
+### Canonical Indices Network: Transactions by Hash and Blocks by Number
 
-To serve the `eth_getTransactionByHash` and `eth_getBlockByNumber` JSON-RPC API endpoints, clients typically build local indexes as they sync the entire chain. The portal network will need to mimic these indices. This can be acheived by generating unique mappings for transactions and blocks, and then pushing these into to portal network.
+The Canonical Indices Network facilitates access to canonical information about transactions and blocks.  This includes:
```

Comment:
Maybe:

The Canonical Indices Network is a pair of indices enabling faster access to transactions and blocks. It includes:
- An index which maps each transaction hash to the block within which the transaction was included
- An index which maps each block number to the hash of the canonical block with that number

Reply:
I think phrasing it this way allows "We refer to the data that backs these two lookups as "Canonical Indices" as they serve as a type of database index for fast retrieval of this information which would otherwise require prohibitively expensive scanning of all historical blocks." to be dropped.